faqs.md

```diff
@@ -12,10 +12,10 @@ The RubyGems development team has gotten a lot of support requests over the
 years, and this is a list of the questions users both new and old that
 frequently pop up.
 
-* [I installed gems with `--user-install` and their commands are not available](#user-install)
-* [How can I trust Gem code that's automatically downloaded?](#security)
-* [Why does `require 'some-gem'` fail?](#require-fail)
-* [Why does require return false when loading a file from a gem?](#require-false)
+- [I installed gems with `--user-install` and their commands are not available](#i-installed-gems-with---user-install-and-their-commands-are-not-available)
+- [How can I trust Gem code that's automatically downloaded?](#how-can-i-trust-gem-code-thats-automatically-downloaded)
+- [Why does `require 'some-gem'` fail?](#why-does-require-some-gem-fail)
+- [Why does require return false when loading a file from a gem?](#why-does-require-return-false-when-loading-a-file-from-a-gem)
```

Comment:
What's the difference between using `-` and `*`? I noticed that other docs are also using `-` for table of contents, so I'm wondering if we can keep the consistency?

Reply:
Yeah sure my bad! Markdownlint changed the format.